lib/tasks/import.rake

```diff
@@ -23,7 +23,7 @@ namespace :import do
       Item.create(id:          row[:id],
                   name:        row[:name],
                   description: row[:description],
-                  price:       row[:unit_price],
+                  unit_price:  row[:unit_price].to_f / 100,
```

Comment:
@agpiermarini Do we need to convert this to a float and divide by 100 here or can we do that conversion later on when we actually need the data? I feel like it's better to keep the unit price in it's raw form as long as possible

Reply:
The prices in the spec harness (for find/find_all) have the decimal point two places to the left (as compared to the raw data), so `Item.find_by({unit_price: "774.84"})` and `Item.where({unit_price: "774.84"})` were both returning nil. My guess is because a person would likely express price in those terms when making a call to the API. I don't like the idea of modifying data on import, but it seems to me to be the cleanest way of resolving the issue. With that said, I am happy to return to it if we can come up with a more appropriate solution.

Reply:
Sounds good!